index.html

```diff
@@ -1,178 +1,236 @@
-        </ul>
-        <button type="button" class="btn-project">See Project</button>
-      </li>
-    </ul>
-  </section>
-  <section class="about-myself">
-    <div class="about-myself-content">
-      <p class="section-title">About Myself</p>
-      <p>
-        Hello I’m a software developer! I can help you build a product ,
-        feature or website Look through some of my work and experience! If you
-        like what you see and have a project you need coded, don’t hestiate to
-        contact me.
-      </p>
-      <p><a href="#" class="main-text">LET’S CONNECT</a></p>
-      <ul>
-        <li>
-          <a href="https://twitter.com/sheyPozo"><img src="img/twitter.png" alt="Icon-Twitter" /></a>
-        </li>
-        <li>
-          <a href="https://www.linkedin.com/in/sheypozo/"><img src="img/linkedin.png" alt="Icon-Linkedin" /></a>
-        </li>
-        <li>
-          <a href="#"><img src="img/M.png" alt="Icon-M" /></a>
-        </li>
-        <li>
-          <a href="https://github.com/sheylaPozo"><img src="img/github.png" alt="Icon-GitHub" /></a>
-        </li>
-        <li>
-          <a href="#"><img src="img/fingers.png" alt="Icon-Fingers" /></a>
-        </li>
-      </ul>
-      <button type="button" class="btn-project">Get my resume</button>
-      <ul class="skills-container">
-        <li>
-          <div class="skills" id="skills-margin">
-            <p>Languages</p>
-            <i class="fas fa-chevron-down"></i>
-          </div>
-          <ul class="lg-dropdown">
-            <li class="lg-dropdown-description">
-              <img src="img/javascript.png" alt="Javascript-icon" />
-              <p>JavaSript</p>
-            </li>
-            <li class="lg-dropdown-description">
-              <img src="img/html.png" alt="HTML-icon" />
-              <p>HTML</p>
-            </li>
-            <li class="lg-dropdown-description">
-              <img src="img/css.png" alt="CSS-icon" />
-              <p>CSS</p>
-            </li>
-          </ul>
-        </li>
-        <li class="skills skills-border">
-          <p>Frameworks</p>
-          <i class="fas fa-chevron-right"></i>
-        </li>
-        <li class="skills skills-border">
-          <p>Skills</p>
-          <i class="fas fa-chevron-right"></i>
-        </li>
-      </ul>
-    </div>
-  </section>
-</body>
-</html>
\ No newline at end of file
+        <form action="https://formspree.io/f/meqvkzyo" method="POST">
+          <input
+            type="text"
+            name="user-name"
+            id="user-name"
+            required
+            placeholder="Name"
+            maxlength="30"
+          />
+          <input
+            type="email"
+            name="user-email"
+            id="user-email"
+            required
+            placeholder="Email"
+          />
+          <textarea
+            name="message"
+            id="message"
+            placeholder="write your message here"
+            maxlength="500"></textarea>
```

Comment:
[OPTIONAL]
I strongly suggest you add " required" attribute to activate the validation on submit for this input field. Currently, the form submits even when nothing is entered in the input field.

Reply:
okii will do :)